v63003/inref/gtm8182.m

```diff
@@ -0,0 +1,33 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+
+gtm8182
+	SET INST1path=$ZTRNLNM("path_INST1","","","","","VALUE")
+	SET INST1gbldir=INST1path_"/mumps.gld"
+
+	SET INST3path=$ZTRNLNM("path_INST3","","","","","VALUE")
+	SET INST3gbldir=INST3path_"/mumps.gld"
+
+	WRITE "    Update the INST1 DB",!
+	SET ^jake="youngest"
+
+	WRITE "    Switch to INST3 gbldir",!
+	SET $ZGBLDIR=INST3gbldir
```

Comment:
I see many sections of the release note not tested. Below are the additional use cases I would like tested.

1. Use extended references (i.e. ^|"INST3.gld"|), not just SET $ZGBLDIR.
2. Test that if the gld has no .repl file mapped, the gtm_repl_instname env var determines which .repl file is picked.
3. Test that if a gld has a .repl file mapped, the mapping overrides the gtm_repl_instname env var.
4. Test that GDE CHANGE -INSTANCE -FILE_NAME="" removes the .repl mapping in the gld file.
5. Try using multiple instances with two global directories where neither has an instance file mapping. You should get an error.
6. Try sharing a region between two instance files and access both instances. You should get an error.
7. Try using two instances with one instance jnlpool not set up. You should get an error.
8. Try updating multiple instances in the same TP transaction. You should get an error.
9. Test that updating multiple instances in the same TP transaction is allowed as long as only non-replicated regions are updated in the second instance.
10. Test $VIEW("JNLPOOL") returns different replication instance file names as you switch across multiple instances.
11. Test that $VIEW("JNLPOOL") after an extended reference continues to correspond to the original instance file and not that of the extended reference whereas for a SET $ZGBLDIR it corresponds to the instance of the new gbldir.
12. Test the EHR application example in the release note (A->B, P->Q) to a reasonable degree. At least, have a process update A and P at the same time to verify things work fine. I will come up with a test case for this bullet.